README.md

```diff
@@ -1,10 +1,28 @@
-| --- | --- | --- | --- | --- | --- | --- | 
 
-The MSAL library for iOS and macOS gives your app the ability to begin using the [Microsoft Identity platform](https://aka.ms/aaddev) by supporting [Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/) and [Microsoft Accounts](https://account.microsoft.com) in a converged experience using industry standard OAuth2 and OpenID Connect. The library also supports [Azure AD B2C](https://azure.microsoft.com/services/active-directory-b2c/) for those using our hosted identity management service.
+| Documentation                  | Code Samples      | Library Reference | Support | Feedback|
+|-------------------------------|---------------------------|-------------------|---------|----------|
+| [MSAL iOS and macOS documentation](https://learn.microsoft.com/en-us/entra/msal/objc/) | &#8226;  [Microsoft Entra ID (workforce samples)](https://learn.microsoft.com/en-us/entra/identity-platform/sample-v2-code?tabs=framework#ios)<br/>&#8226; [Microsoft Entra External ID (customer samples)](https://learn.microsoft.com/en-us/entra/external-id/customers/samples-ciam-all?tabs=apptype#mobile)          | [ SDK reference](https://azuread.github.io/microsoft-authentication-library-for-objc/)             | [Get support](README.md#community-help-and-support) | [Feedback](https://forms.office.com/r/xuBV0CzEih) |
+
+
+The Microsoft Authentication Library (MSAL) for iOS and macOS is an Auth SDK that can be used to seamlessly integrate authentication into your apps using industry standard OAuth2 and OpenID Connect. It allows you to sign in users or apps with Microsoft identities. These identities include Microsoft Entra ID work and school accounts, personal microsoft accounts, social accounts, customer accounts, and more. Using MSAL for iOS and macOS, you can acquire security tokens from the Microsoft identity platform to authenticate users and access secure web APIs for their applications.
+
+MSAL for iOS and macOS supports multiple authentication scenarios, such as single sign-on (SSO), conditional access, and brokered authentication. The library also provides native authentication APIs that allow applications to implement a native experience with end-to-end customizable flows in their mobile applications. 
+
+## Get started
+
+To use MSAL iOS and macOS in your application, you need to register your application in the Microsoft Entra Admin center and configure your project. Since the SDK supports both browser-delegated and native authentication experiences, follow the steps in the one of these quickstarts based on your scenario.
```

Comment:
Native Auth code will not be available to macOS, can we specify this note somewhere in this doc?